src/main/daml/Daml/Finance/Instrument/Swap/Fpml/Instrument.daml

```diff
@@ -56,7 +57,7 @@ template Instrument
     let instrumentKey = InstrumentKey with depository; issuer; id; version
 
     interface instance Claim.I for Instrument where
-      view = Claim.View with instrument = instrumentKey; acquisitionTime = dateToDateClockTime issueDate
+      view = Claim.View with instrument = instrumentKey; acquisitionTime = dateToDateClockTime (date 1970 Jan 01) -- sufficiently early to guarantee that any time-shifted fixings are observed on the correct date
```

Comment:
maybe use `daysSinceEpochToDate 0` instead

Reply:
Yes, that looks better, thanks!